src/mds/Server.cc

```diff
@@ -362,8 +362,9 @@ void Server::dispatch(const cref_t<Message> &m)
     handle_peer_request(ref_cast<MMDSPeerRequest>(m));
     return;
   default:
-    derr << "server unknown message " << m->get_type() << dendl;
-    ceph_abort_msg("server unknown message");  
+    dout(10) << "server unknown message: " << m->get_type() << ", therefore closing session and blocklisting client " << session->get_client() << dendl;
```

Comment:
Maybe we should set the loglevel to `-1` instead ?

Reply:
Yeah, as per our conversation, I've reverted back to `derr`.